src/lib.rs

```diff
@@ -221,7 +224,7 @@ mod test {
                 }
             });
         }
-        sleep_ms(1000);
+        sleep_ms(1024);
```

Comment:
I had it down to 10ms, but it was broken on some machines. then I choos a geeky number.